tests/integration/istioio/security/authz_http/setup.sh

```diff
@@ -35,3 +35,21 @@ spec:
     tls:
       mode: ISTIO_MUTUAL
 EOF
+
+# Apply the resources.
+pushd ${TOP}/src/istio.io/istio
+kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
```

Comment:
Should these match the @samples/bookinfo/platform/kube/bookinfo.yaml@ style so the proper links get generated? 
https://istio.io/about/contribute/creating-and-editing-pages/#links-to-github-files

Reply:
Nathan and I chatted about that yesterday. This is another thing we'll address after this goes in and we have the pipeline working end to end.

Reply:
Ok. The functionality to support the @ style is already in here, but is being removed in this PR.

Reply:
@brian-avery ah that's the context I was missing, thanks! I didn't fully appreciate what you were doing with the syntax (I haven't written any content on istio.io).

I'll try to rework things a bit this morning ... I don't think it will be hard.

Reply:
No problem. :)